jenkins/automation/s3server/s3_splunk_testcase.groovy

```diff
@@ -0,0 +1,82 @@
+pipeline { 	 
+    agent {
+		node {
+            // Agent created with 4GB ram/16GB memory in EOS_SVC_RE1 account 
+			label "s3-compatibility-test-ceph-rhev-7.9"
+            // Use custom workspace for easy troublshooting
+            customWorkspace "/root/compatability-test/${INTEGRATION_TYPE}"
+		}
+	}
+
+	options {
+        timeout(time: 240, unit: 'MINUTES')
+        timestamps()
+        ansiColor('xterm')
+        buildDiscarder(logRotator(daysToKeepStr: '5', numToKeepStr: '5')) 
+    }
+
+    parameters {
+        string(name: 'RGW_PORT', defaultValue: '30080', description: 's3-test rgw port', trim: true)
+        string(name: 'RGW_SERVICE_IP', defaultValue: '', description: 's3-test rgw ip address', trim: true)
+        string(name: 'S3_TEST_REPO', defaultValue: 'https://github.com/splunk/s3-tests', description: 's3-test splunk repo', trim: true)
+        // we are using specific revision of 'https://github.com/splunk/s3-tests' for our tests  - default
+        string(name: 'S3_TEST_REPO_REV', defaultValue: '3dc9362b1d322a59bd4e8f207d5a94070502b78b', description: 's3-test repo revision', trim: true)
+        choice(name: 'INTEGRATION_TYPE', choices: [ "splunk"], description: 'S3 Integration Type') 
+	}
+
+    environment {
+        // This config file used for splunk compatibility tests
+        S3_TEST_CONF_FILE = "${INTEGRATION_TYPE}_${BUILD_NUMBER}.conf"
+    }
+
+	stages {
+        // Update test config for s3server auth credentials
+        stage ('Update Test Config') {    
```

Comment:
Please change the stage name to "Execute Test cases"

Reply:
Changed stage name